subjects/cameltosnakecase/README.md

```diff
@@ -46,9 +47,10 @@ and the output should be:
 
 ```console
 $ go run .
-Hello_World
-helloWorld
-Camel_To_Snake_Case
+HelloWorld
```

Comment:
This is camel as well so it should be changed...

Reply:
Yes, it is a camel case but an Upper camel case, and I use only the lower camel case in the subject.

Reply:
then you must e specific in the readme.
Dont use "camelCase" and instead use "lowerCamelCase" otherwise it will create some confusion...

Reply:
Modified.

Reply:
Still this output is wrong...
The "HelloWorld" is CamelCase so why isn't the output "Hello_World"?

Reply:
@MSilva95 I'm sure that the output, in this case, is the right one.
In the subject, I use only the Lower camel case. 
> Lower camel case is the practice of writing phrases without spaces or punctuation; it indicates the separation of one word or more with a single capitalized letter between each word except the first one. Snake case is a style of writing in which each space is replaced by an underscore (_) character.
 
That is why the output of this case is `HelloWorld ` and not Hello_Word

Reply:
It is not specified anywhere that only lower camel is to be considered, actually it is asked for the student to
"Write a function that converts a string from camelCase to snake_case."

In my interpretation you are asking for any kind of camel, so that one should be a wrong output.
Also in my opinion it is a better exercise if it accepts both but that is up to you to decide how do you want this exercise to work.
In case you decide to have just the lowerCamelCase, please don't use this as it is misleading:
```
Write a function that converts a string from camelCase to snake_case.
- If the string is not `camelCase`, return the string unchanged.
- If the string is `camelCase`, return the `snake_case` version of the string.
```

Also there are some rule not mentioned in your readme like:

- The word does not end on a capitalized letter (CamelCasE).
- No two capitalized letters shall follow directly each other (CamelCAse).

Reply:
Good point, I change the solution and the readme to manage the cases of Lower and Upper camel cases.